proto/buf/alpha/studio/v1alpha1/invoke.proto

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+// Because the request agent needs to be able to receive simple CORS requests,
+// we're limited to the text/plain Content-Type. This isn't suported out of the
+// box in gRPC-Web, so for now we avoid using it and do plain POST with base64
+// proto request/response messages.
+//
+// We may explore websocket transports later, at which point we should define
+// services methods here.
+package buf.alpha.studio.v1alpha1;
```

Comment:
can this comment include what this package is for?

Reply:
Added a better description of what these envelope messages are for.